ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -99,13 +108,19 @@ public ClaimData from(CCDClaim ccdClaim) {
 
         List<Party> claimants = ccdClaim.getClaimants()
             .stream()
+            .filter(c -> c.containsKey("value"))
             .map(this::valueFromMap)
+            .map(this::toJson)
+            .map(s -> this.fromJson(s, CCDParty.class))
             .map(partyMapper::from)
             .collect(Collectors.toList());
 
         List<TheirDetails> defendants = ccdClaim.getDefendants()
             .stream()
+            .filter(c -> c.containsKey("value"))
             .map(this::valueFromMap)
+            .map(this::toJson)
+            .map(s -> this.fromJson(s, CCDParty.class))
```

Comment:
defendants are stored as array in ccd and it generates an id for each array element, which is UUID, and it can't mapped to CCDParty, so need to extract object by filtering out id and map again